.env.example

```diff
@@ -58,12 +58,12 @@ PGSSLMODE=
 # @see: https://github.com/calendso/calendso/issues/263
 # @see: https://next-auth.js.org/configuration/options#nextauth_url
 # Required for Vercel hosting - set NEXTAUTH_URL to equal your NEXT_PUBLIC_WEBAPP_URL
-NEXTAUTH_URL='http://localhost:3000'
+NEXTAUTH_URL='http://app.cal.local:3000'
```

Comment:
Had to use a different URL than "localhost" to play with cookie sessions between different subdomains. We should make everyone use this new scheme by default, it's needed to make orgs work locally.